EG23.tex

```diff
@@ -333,18 +361,21 @@ \subsubsection{Case 2: On-Demand Separations}
 \subsection{Fuel Fabrication}
 
 The UOX fuel fabrication is handled by one enrichment facility, the properties
-of this enrichment facilities are summarized in Table \ref{tab:enrich_1}.
+of this enrichment facilities are summarized in Table \ref{tab:enrich_1}. As
+explain the specifcication of the enrichment facility have been taken from the
+R. Carlsen calculation and might differ from the exact EG01-23 transition
+specifications. 
```

Comment:
I did't have the detailed specifications on those points and used Robert definition.